### PR TITLE
Return TWave on multiplication with scalar

### PR DESCRIPTION
**Pull request by darshanime**
On trying to multiply `TWave` object from `sympy.physics.optics` module with a scalar, `sympy.core.mul.Mul` was returned. 
PR to return `TWave` with amplitude scaled.

### CODE REVIEW

File: sympy/physics/optics/waves.py [hunk 1]
```diff
@@ -71,6 +71,12 @@ class TWave(Expr):
     299792458*m/(n*s)
     >>> w3.angular_velocity
     2*pi*f
+    >>>w4 = 2*w1
+    >>>w4.amplitude
+    2*A1
+    >>>w4.phase
```
Comment (debugger22) [inline sympy/physics/optics/waves.py]: Put a space after `>>>`

**Comment by debugger22** [comment]
Make a single final commit after making changes.

**Comment by darshanime** [comment]
I messed up the rebase. Issued another PR with the changes.
Sorry for the inconvenience.

**Comment by debugger22** [comment]
Replaced by https://github.com/sympy/sympy/pull/8895

**Pull request closed without merging**